zkevm-circuits/src/bytecode_circuit/bytecode_unroller.rs

```diff
@@ -92,6 +102,28 @@ impl<F: Field> Config<F> {
             push_rindex_inv,
         );
 
+        // Does the current row have bytecode field tag == Length?
+        let is_row_tag_length = |meta: &mut VirtualCells<F>| {
```

Comment:
one potential optimization is that we just use 0/1 to serve as a selector for `length` or `byte`. So we can reduce the degree from 2 to 1

Reply:
In that case, `padding` must be changed to `2` since it's used in the `tag` as well. At the moment `padding` is represented as `0`, but this point makes sense and I'll make the update.

Reply:
ah, I see. In that case, I think either way looks good to me.